README.md

```diff
@@ -35,183 +27,57 @@ OpenTTD is licensed under the GNU General Public License version 2.0,
-will not be able to reproduce the desync and thus will be unable to fix it.
-
-## 3.0) Supported platforms
-
-OpenTTD has been ported to several platforms and operating systems. It should
-not be very difficult to port it to a new platform. The currently working
-platforms are:
+## 1.1) Downloading OpenTTD
+
+OpenTTD can be downloaded from the [official OpenTTD website](https://www.openttd.org/).
+
+Both 'stable' and 'nightly' versions are available for download.
+
+Most people will want the 'stable' version.
+
+The 'nightly' version includes the latest code changes and may be less reliable.
+
+On some platforms OpenTTD will also be available via your OS package manager or a similar service.
+
+
+## 1.2) OpenTTD gameplay manual
+
+OpenTTD has a [community-maintained wiki](https://wiki.openttd.org/), including a gameplay manual and tips.
+
+
+## 1.3) Supported platforms
+
+OpenTTD has been ported to several platforms and operating systems.
+
+The currently working platforms are:
 
 - FreeBSD (SDL)
 - Haiku (SDL)
-- Linux (SDL or Allegro)
+- Linux (SDL)
 - macOS (universal) (Cocoa video and sound drivers)
 - OpenBSD (SDL)
 - OS/2 (SDL)
-- Windows (Win32 GDI (faster) or SDL or Allegro)
+- Windows (Win32 GDI (faster) or SDL)
 
-## 4.0) Installing and running OpenTTD
+### 1.3.1) Legacy support
+Platforms, languages and compilers change.
+We'll keep support going on old platforms as long as someone is interested in supporting them, except where it means the project can't move the project forward to keep up with language and compiler features.
 
-Installing OpenTTD is fairly straightforward. Either you have downloaded an
-archive which you have to extract to a directory where you want OpenTTD to
-be installed, or you have downloaded an installer, which will automatically
-extract OpenTTD in the given directory.
+We'll also do our best to maintain compatibility with old savegames, except where it becomes impractical.
 
-OpenTTD looks in multiple locations to find the required data files (described
-in section 4.2). Installing any 3rd party files into a 'shared' location has
-the advantage that you only need to do this step once, rather than copying the
-data files into all OpenTTD versions you have.
 
-Savegames, screenshots, etc are saved relative to the config file (openttd.cfg)
-currently being used. This means that if you use a config file in one of the
-shared directories, savegames will reside in the save/ directory next to the
-openttd.cfg file there.
+## 1.4) Installing and running OpenTTD
 
-If you want savegames and screenshots in the directory where the OpenTTD binary
-resides, simply have your config file in that location. But if you remove this
-config file, savegames will still be in this directory (see notes in
-section 4.2 'OpenTTD directories')
+OpenTTD is usually straightforward to install.  Installing varies by platform, and the wiki [includes an installation guide](https://wiki.openttd.org/Installation).
```

Comment:
Uhm, what?!?

Reply:
That's a very expressive and funny comment, but if this line is wrong or confusing, more specific reaction would be useful to improve it...

Reply:
"is usually straightforward to install." is tautological nonsense that has no place in a readme. people who have no trouble installing won't ever read this sentence (because why would you read a readme if you don't have a problem), and people who have problems installing will be mocked by this sentence.

Reply:
Are the wiki installation guides sufficient for problem solving?  

Or to ask the question another way:
* option A is simply to reword this to address your point (which I agree with)
* option B is to identify that we need to provide trouble-shooting guidance within the README (or linked from it)